File: pset1/mario.c
// Solution to CS50's Fall 2019 Pset1 / mario.c by Nicolas Matiz.

#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
   
    do{
        height = get_int("height: ");
    }
    while(height > 8 || height < 1);

    for(int i = 0; i < height; i++){
        
        for(int j = 0; j < height-1-i; j++){
            printf(" ");
        }
        for(int k = 0; k < i+1; k++){
            printf("#");
        }
        printf("\n");
    }
    
    return 0;
}

File: pset6/mario.py
# Solution to CS50's Fall 2019 Pset6 / mario.py by Nicolas Matiz.

from cs50 import get_int

height = 0
while height > 8 or height < 1:
    height = get_int("height: ")

for i in range(height):
    for j in range(height-1-i):
        print(" ",end="")
    for k in range(i+1):
        print("#",end="")
    print("")
